src/components/tutorials-landing-view/collection-content-card-link/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { ProductUsed, TutorialLite } from 'lib/learn-client/types'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import { type CollectionContentCardLinkProps } from '../types'
+import ContentCardLink from '../content-card-link'
+import { BADGE_ICON_MAP, PRODUCT_SLUGS_TO_HEADER_IMAGES } from './constants'
+
+const CollectionContentCardLink = ({
+	productSlug,
```

Comment:
We could calculate this from the first segment of the collection slug!

```ts
import { normalizeSlugForDevDot } from 'lib/tutorials/normalize-product-like-slug'
const productSlug = normalizeSlugForDevDot(collection.slug.split('/')[0])
```

Reply:
Updated in:

- https://github.com/hashicorp/dev-portal/pull/1957/commits/c6a89204356847b55f0daa69b5dcb4b2933510a4
- https://github.com/hashicorp/dev-portal/pull/1957/commits/ed3313de84f8db3a19906b30eefd058825f17fe9